cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/MaintenancePlanSerdeTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
```

Comment:
Nit: `2020` -> `2021`  -- i.e. by convention new files use the year of creation.

Reply:
Nit: This feedback seems missed.